addon/components/nrg-validation-component.js

```diff
@@ -35,6 +43,21 @@ export default class NrgValidationComponent extends Component {
     }
   }
 
+  get useDefaultValue() {
+    return this.args.useDefaultValue ?? false;
```

Comment:
```suggestion
    return this.args.defaultValue ?? this.args.useDefaultValue ?? false;
```

I feel like if you pass in a defaultValue, it should always be used.

Reply:
What if the default value is null or undefined, but we want to use it?

Reply:
Seems obscure, but I'll accept that.